### ISSUE TITLE
Tags inside list elements are not handled

### ISSUE THREAD

**Issue by pchupik** [commonsguy/cwac-richedit#17]
If we have 
```html
<ul><li>The <b>quick</b> brown fox</li><li>jumped over the lazy dog.</li></ul>
```
after fromXhtml( ) and back toXhtml( ) result will be
```html
<ul><li>The quick brown fox</li><li>jumped over the lazy dog.</li></ul>
```
It is expected tags inside lists should be saved.

**Comment by commonsguy** [comment]
Yes, I can reproduce this problem. I will aim to get it fixed in the next update. Thanks!

**Comment by commonsguy** [comment]
Fixed in 0.5.1. Thanks!
